modules/openvdb/11.0.0.bcr.1/MODULE.bazel

```
@@ -0,0 +1,12 @@
+"""https://www.postgresql.org/"""
```

Postgresql?

---

Ah! Confused this with my other pull-request.
```suggestion
"""https://www.openvdb.org/"""
```

Thanks!